locust/event.py

```diff
@@ -179,3 +203,29 @@ def __init__(self):
         for name, value in self.__annotations__.items():
             if value == EventHook:
                 setattr(self, name, value())
+
+        setattr(
+            self, "request_success", DeprecatedEventHook("request_success event deprecated. Use the request event.")
+        )
+        setattr(
+            self, "request_failure", DeprecatedEventHook("request_failure event deprecated. Use the request event.")
+        )
+
+        def on_request(request_type, name, response_time, response_length, exception, context, **kwargs):
```

Comment:
this method could be named something more like "fire_deprecated_request_handlers"